client/html/src/Client/Html/Checkout/Standard/Process/Standard.php

```diff
@@ -266,15 +270,25 @@ public function process()
 			$orderCntl = \Aimeos\Controller\Frontend\Factory::createController( $context, 'order' );
 			$basketCntl = \Aimeos\Controller\Frontend\Factory::createController( $context, 'basket' );
 
-			$basketCntl->get()->setCustomerId( $context->getUserId() );
 
-			parent::process();
+			if ( $view->param( 'cs_order', null ) !== null ) {
```

Comment:
This is likely wrong because if the payment data shouldn't be captured locally, only the order is created but the payment provider isn't called. Correct should be:
```
if ( $view->param( 'cp_payment', null ) === null )
```
for creating the order and no elseif branch to always call the payment provider.